packages/calcite-components/src/components/link/link.tsx

```diff
@@ -151,13 +151,12 @@ export class Link extends LitElement implements InteractiveComponent {
     );
 
     const DynamicHtmlTag =
-      childElType === "span"
-        ? (literal`span` as unknown as "span")
+      childElType === "button"
+        ? (literal`button` as unknown as "button")
         : (literal`a` as unknown as "a");
-    const role = childElType === "span" ? "link" : null;
-    const tabIndex = childElType === "span" ? 0 : null;
+    const tabIndex = childElType === "button" ? 0 : null;
     /* TODO: [MIGRATION] This used <Host> before. In Stencil, <Host> props overwrite user-provided props. If you don't wish to overwrite user-values, replace "=" here with "??=" */
-    this.el.role = "presentation";
+    this.el.role = "presentation"; // todo: doesn't this make this component not accessible?
```

Comment:
@geospatialem would this be problematic as it would make the whole component non accessible for screen readers?

Looks like it was added here: https://github.com/Esri/calcite-design-system/pull/1337 cc @jcfranco

Reply:
I believe it just removes that specific element from the AT, but wouldn't impact the roles of internal, children. Looks like it was added in the early component days, and might not even be needed since the internal elements provide the semantics. 🤔 

FWIW, both Chrome and Firefox show the proper roles of the internal link:

![Screenshot 2025-05-21 at 10 33 42 AM](https://github.com/user-attachments/assets/c1fbafa4-0798-4da0-b980-6bef6af6b6a3)

![Screenshot 2025-05-21 at 10 33 23 AM](https://github.com/user-attachments/assets/30085095-1773-4cf4-8594-a90238832ed7)

Reply:
@geospatialem can you advise? Should we leave it for now or remove it?

Reply:
@driskull Thanks for the re-ping, as Franco [mentions above](https://github.com/Esri/calcite-design-system/pull/12191#discussion_r2100837104), this doesn't seem to impact the role with AT. Regardless of a `href` value the component's context of link is still provided. So we can leave it for now 👍🏻